zoo/policies/__init__.py

```diff
@@ -100,3 +104,86 @@ def human_keyboard_entrypoint(*arg, **kwargs):
 
 
 register(locator="human-in-the-loop-v0", entry_point=human_keyboard_entrypoint)
+
+
+from smarts.env.multi_scenario_env import resolve_agent_interface
```

Comment:
Is there a reason this isn't at the top of the file? If not, it should be moved.